src/turtle.rs

```diff
@@ -1156,4 +1158,26 @@ mod tests {
         assert_eq!(turtle.position()[1].round(), 100.0);
         assert_eq!(turtle.heading(), 51.0);
     }
+
+    #[test]
+    fn turn_towards() {
+        let mut turtle = Turtle::new();
+        turtle.set_speed("instant");
```

Comment:
This line shouldn't be necessary since we disable all animations during tests. Try running the tests with `cargo test --features test` and let me know if for some reason keeping this line actually makes a difference.